tests/shared/models/cb-admin.test.js

```diff
@@ -0,0 +1,144 @@
+const test = require('tape');
+const pg = require('pg');
+const { getConfig } = require('../../../config');
+const { refresh: refreshDB } = require('../../../db/scripts');
+const cbAdmin = require('../../../react-backend/shared/models/cb-admin');
+
+const config = getConfig(process.env.NODE_ENV);
+
+test('Db Query | cbAdmin ', async tape => {
+  const client = new pg.Client(config.psql);
+  await client.connect();
+
+  tape.test('cbAdmin.getFeedback | existing cb_id', async t => {
+    try {
+      await refreshDB();
+
+      const actualAll = await cbAdmin.getFeedback(client, { cbId: '3' });
+      const expectedAll = [
+        {
+          feedback_score: -1,
+          feedback_date: new Date('2017-09-09T17:45:00.000Z'),
+        },
+        {
+          feedback_score: 0,
+          feedback_date: new Date('2017-10-09T17:45:00.000Z'),
+        },
+        {
+          feedback_score: 1,
+          feedback_date: new Date('2017-11-09T17:23:00.000Z'),
+        },
+        {
+          feedback_score: 1,
+          feedback_date: new Date('2017-12-09T17:12:00.000Z'),
+        },
+      ];
+
+      t.deepEquals(
+        actualAll,
+        expectedAll,
+        'cbAdmin.getFeedback returns all cbs feedback without dates specified'
+      );
+      const sinceDate = new Date('2017-11-09 17:23:00+00');
```

Comment:
Split these with/without dates into separate test cases.

Reply:
How come? I was lumping them together under the idea that they were all testing different edge cases of the same function.

Reply:
I guess it's a preference thing, but IMO it's hard to read and maintain long test cases.